lntest/node.go

```diff
@@ -250,9 +250,10 @@ type HarnessNode struct {
 
 	invoicesrpc.InvoicesClient
 
-	// RouterClient cannot be embedded, because a name collision would occur
-	// on the main rpc SendPayment.
-	RouterClient routerrpc.RouterClient
+	// RouterClient and WalletKitClient cannot be embedded, because a name
+	// collision would occur with LightningClient.
```

Comment:
Is this also true for `WalletKitClient`? `LightningClient` and `RouterClient` both have a `SendPayment` call, that is where the collision comes from.

Reply:
Yes, the collision happens with `EstimateFee`.

Reply:
Ah, ok.